isubit/luggage_announcements
# LUGG-752 Remove slideshow setting from flexslider

Turned off 'Slideshow' mode to stop flexslider from rotating automatically, added pause and play buttons, and added "pause on hover" functionality.

Testing:
1. Pull down my changes
2. Run drush fra -y and drush cc all
3. See if flexslider on front page of site rotates automatically or if it requires user input (either swiping on a touchscreen, clicking the arrows, or using the left/right keys).
